### ISSUE TITLE
Update the API

### ISSUE THREAD

**Issue by TrooperZ** [billy-yoyo/RainbowSixSiege-Python-API#82]
Hey, so with the new season North Star coming out with Thunderbird, I was wondering if the API could get updated with Flores and Thunderbird, and also add the unranked playlist stats?

Also I  get the `aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host game-rainbow6.ubi.com:443 ssl:default [getaddrinfo failed]` error when getting operators or ranked regions

**Comment by LaT-Tee** [comment]
Yes, but if I try to run my own program on another VPS, it still works. It seems like Ubisoft only removes its DNS record, the servers are still online.
So I think if I can connect game-rainbow6.ubi.com by customize DNS record, this problem may be solved.

**Comment by PrestonZ2000** [comment]
It seems like ubisoft changed to another domain. [https://www.ubisoft.com/en-us/game/rainbow-six/siege/stats/summary](url)
